io/subsystem/src/main/java/org/wildfly/extension/io/IOSubsystemAdd.java

```diff
@@ -23,11 +23,6 @@
  */
 class IOSubsystemAdd extends AbstractAddStepHandler {
 
-    static final IOSubsystemAdd INSTANCE = new IOSubsystemAdd();
```

Comment:
My question might be bogus, since I'm not familiar with the latest improvements in WF code... but anyway, why exactly are we removing INSTANCE here?

Reply:
Because it is never referenced, nor would it need to be.